### ISSUE TITLE
EPUBs without a root dir fail to ingest

### ISSUE THREAD

**Issue by SMaxOwok** [ManifoldScholar/manifold#1684]
```
No Strategy found for /Users/smyret/src/manifold/api/tmp/ingestion/8abc34c6-98db-40be-b3ad-7e9944937840/source/META-INF
```

```
[1] pry(#<Ingestions::Strategies::Epub>)> context.sources
13:11:58 cable.1     | => ["source/META-INF/container.xml"]

13:12:24 cable.1     | [2] pry(#<Ingestions::Strategies::Epub>)> File.join(inspector.rel_source_root, "/META-INF")
13:12:24 cable.1     | => "source/META-INF/META-INF"
```

Seems as though 
```
        def rel_source_root
          context.rel(context.source_root)
        end
```

in the epub inspector returns the `META-INF` directory if the source files aren't in a parent dir.

**Comment by SMaxOwok** [comment]
Actually, likely related to #1683

**Comment by SMaxOwok** [comment]
Related to #1682 and will be solved by #1685
